linchpin/provision/roles/openshift/tasks/provision_openshift.yml

```diff
@@ -82,23 +81,28 @@
 
 - name: fail when pod_output is failure
   fail:
-    msg: "Multiple pods in pending/failure state please adjust the timeout value and retry"
-  when: state == "present" and pod_output == 'Failure' 
+    msg: "Multiple pods in pending/failure \
```

Comment:
Are you sure this works the way you expect it to?  Generally in yaml the way to split a string across multiple lines is this:
```
msg: >
  Multiple pods in pending/failure state, please adjust
  the timeout value and retry
```

Reply:
@14rcole I tested it with a simple debug msg task. I went fine. 
I can change it to the above format if its the general standard.

Reply:
I did the above for the other modules, but we can standardize on whichever one you prefer

Reply:
@14rcole When I use the above syntax ansible lint was failing.